flowetl/flowetl/flowetl/operators/staging/sql/append_sightings_to_main_table.sql

```diff
@@ -0,0 +1,12 @@
+BEGIN;
+
+-- Partitions are inclusive on lower, exclusive on upper
+ALTER TABLE reduced.sightings
+ATTACH PARTITION reduced.sightings_{{ds_nodash}} FOR VALUES FROM (date('{{ds_nodash}}')) TO (date('{{ds_nodash}}')+ 1) ;
+
+ALTER TABLE reduced.sightings_{{ds_nodash}}
+DROP CONSTRAINT sightings_{{ds_nodash}}_check;
+
+ANALYZE reduced.sightings;
```

Comment:
I would split this out I think, doesn't need to be in the same transaction.

Reply:
You reckon different `begin .. commit` block, or separate operator?